airflow/providers/snowflake/operators/snowflake.py

```diff
@@ -539,16 +539,16 @@ def poll_on_queries(self):
     def execute_complete(self, context: Context, event: dict[str, str | list[str]] | None = None) -> None:
         """
         Callback for when the trigger fires - returns immediately.
-        Relies on trigger to throw an exception, otherwise it assumes execution was
-        successful.
+
+        Relies on trigger to throw an exception, otherwise it assumes execution was successful.
         """
         if event:
             if "status" in event and event["status"] == "error":
                 msg = f"{event['status']}: {event['message']}"
                 raise AirflowException(msg)
             elif "status" in event and event["status"] == "success":
                 hook = SnowflakeSqlApiHook(snowflake_conn_id=self.snowflake_conn_id)
-                query_ids = cast(List[str], event["statement_query_ids"])
+                query_ids = cast(list[str], event["statement_query_ids"])
```

Comment:
This change would fail at runtime on 3.8.

Reply:
It did actually :)

Reply:
Yeah, `ruff` did that as an autofix and I thought I caught them all, I missed this one.  Good catch.

Reply:
FWIW I figured this one out; I was running `ruff` manually one directory at a time to save time and keep my fix list manageable but I accidentally used the autofix flag on it.   Shouldn't happen again.